nessie/jobs/create_lrs_glue_jobs.py

```diff
@@ -0,0 +1,70 @@
+"""
+Copyright ©2018. The Regents of the University of California (Regents). All Rights Reserved.
+
+Permission to use, copy, modify, and distribute this software and its documentation
+for educational, research, and not-for-profit purposes, without fee and without a
+signed licensing agreement, is hereby granted, provided that the above copyright
+notice, this paragraph and the following two paragraphs appear in all copies,
+modifications, and distributions.
+
+Contact The Office of Technology Licensing, UC Berkeley, 2150 Shattuck Avenue,
+Suite 510, Berkeley, CA 94720-1620, (510) 643-7201, otl@berkeley.edu,
+http://ipira.berkeley.edu/industry-info for commercial licensing opportunities.
+
+IN NO EVENT SHALL REGENTS BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT, SPECIAL,
+INCIDENTAL, OR CONSEQUENTIAL DAMAGES, INCLUDING LOST PROFITS, ARISING OUT OF
+THE USE OF THIS SOFTWARE AND ITS DOCUMENTATION, EVEN IF REGENTS HAS BEEN ADVISED
+OF THE POSSIBILITY OF SUCH DAMAGE.
+
+REGENTS SPECIFICALLY DISCLAIMS ANY WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE. THE
+SOFTWARE AND ACCOMPANYING DOCUMENTATION, IF ANY, PROVIDED HEREUNDER IS PROVIDED
+"AS IS". REGENTS HAS NO OBLIGATION TO PROVIDE MAINTENANCE, SUPPORT, UPDATES,
+ENHANCEMENTS, OR MODIFICATIONS.
+"""
+
+
+"""Logic for transforming LRS Caliper statements."""
+
+
+from flask import current_app as app
+from nessie.externals import glue
+from nessie.jobs.background_job import BackgroundJob
+
+
+class CreateLrsGlueJobs(BackgroundJob):
+
+    def run(self):
+        app.logger.info('Create Glue job to process on LRS incrementals')
+        if self.create_lrs_caliper_relationalize_job():
+            return True
+        else:
+            return False
+
+    def create_lrs_caliper_relationalize_job(self):
+        job_name = app.config['LRS_CANVAS_GLUE_JOB_NAME']
+        glue_role = app.config['LRS_GLUE_SERVICE_ROLE']
+        allocated_capacity = 2  # app.config['LRS_CANVAS_GLUE_JOB_CAPACITY']
+        timeout = 20  # app.config['LRS_CANVAS_GLUE_JOB_TIMEOUT']
```

Comment:
@sandeepmjay Are these commented-out configs from work in progress?

Reply:
While creating the jobs I want to force the defaults. During job run the parameters can be changed and can refer the config file. It is easy to set these numbers to something extravagant and spin up resources which can lead to added costs.